source/daplink/drag-n-drop/iap_flash_intf.c

```diff
@@ -244,11 +244,10 @@ static error_t program_page(uint32_t addr, const uint8_t *buf, uint32_t size)
     return ERROR_SUCCESS;
 }
 
-static error_t erase_sector(uint32_t addr)
+static error_t erase_sector(uint32_t sector)
```

Comment:
The definition at the top of the file uses a sector number instead of address. I ran into this issue because this erase_sector function was taking an address and the target_flash_erase_sector function was taking a sector number.